sprints/templates/base.html

```diff
@@ -59,13 +59,10 @@
                 <a class="nav-link" href="{% url 'account_logout' %}">{% trans "Sign Out" %}</a>
               </li>
             {% else %}
+              {% load socialaccount %}
               <li class="nav-item">
                 {# URL provided by django-allauth/account/urls.py #}
-                <a id="sign-up-link" class="nav-link" href="{% url 'account_signup' %}">{% trans "Sign Up" %}</a>
-              </li>
-              <li class="nav-item">
-                {# URL provided by django-allauth/account/urls.py #}
-                <a id="log-in-link" class="nav-link" href="{% url 'account_login' %}">{% trans "Sign In" %}</a>
+                <a id="log-in-link" class="nav-link" href="{% provider_login_url 'google' %}">{% trans "Sign In" %}</a>
```

Comment:
@Agrendalath, is there a way to create a login page that prompts for username/email and password and allows optionally using Google for authentication - kinda like how LMS login page is?

Reply:
@Agrendalath, I tested this and it works! Can we disable registration since this will be exposed to the internet? I am not sure if it is possible - but can we have the registration done solely through Google and the actual login be done via either? This would require the ability to set or change the password for a user created via Google. This is just a suggestion and can be worked on in a separate ticket.

Also, a minor nit (can be fixed during further work on this project) that I noticed was that on the `/users/~update/` page, the `Update` button doesn't have any styling.

Reply:
@lgp171188, sure, we can just set up `DJANGO_ACCOUNT_ALLOW_REGISTRATION` to `False` on prod. We also have `ACCOUNT_ALLOWED_EMAIL_DOMAINS` now, so I don't think that open registration makes a huge difference here.

> This would require the ability to set or change the password for a user created via Google.
Makes sense. Currently it's possible to use "Forgot password" button for this.

The `/users/~update/` page will be removed in BB-1244 as we won't be using it.